profiling/src/exporter/mod.rs

```diff
@@ -209,13 +187,37 @@ impl ProfileExporter {
                     ("aas.site.type", aas_metadata.get_site_type()),
                     ("aas.subscription.id", aas_metadata.get_subscription_id()),
                 ];
-                aas_tags
-                    .into_iter()
-                    .for_each(|(name, value)| form.add_text(name, value));
+                aas_tags.into_iter().for_each(|(name, value)| {
+                    tags_profiler.push_str(Tag::new(name, value).unwrap().as_ref());
```

Comment:
On this particular unwrap, the tag hasn't been validate yet, has it? This unwrap could panic if for some reason it contains something that isn't considered a valid tag, right?

Reply:
Oh you are right.
Will fix it.